src/wp-includes/class-wp-theme-json.php

```diff
@@ -1053,11 +1053,13 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 			}
 			$stylesheet .= $this->get_block_classes( $style_nodes );
 		} elseif ( in_array( 'base-layout-styles', $types, true ) ) {
-			$root_selector    = static::ROOT_BLOCK_SELECTOR;
-			$columns_selector = '.wp-block-columns';
+			$root_selector          = static::ROOT_BLOCK_SELECTOR;
```

Comment:
Would we need a `@since` annotation in the function PHPDoc block? E.g., 

`* @since 6.3.0 Add fallback layout styles for Post Template when block gap support isn't available.` ??

Reply:
Ah good point, yes, I'll add that in!